src/scratchpad.js

```diff
@@ -132,7 +132,26 @@ export default function Scratchpad (DOM, props) {
     }
   };
 
+  const clientWidth$ = DOM.observable.pluck('clientWidth');
+
+  const mouseDowns$ = DOM.select('.handler').events('mousedown');
```

Comment:
Instead of calling it `mouseDowns$`, just call it `mouseDown$`.

In the same way that if you have an array of `books` each item is a `book`, if you have a stream of `book$` each item is a `book`.

The rationale is that if you want to have a stream of arrays of books, it would be `books$` because each item is an array of books.